demo-app/src/open-truss/configs/7-render-frames-demo.yaml

```diff
@@ -0,0 +1,19 @@
+workflow:
+  version: 1
+  id: 7-render-frames-demo-unique-id
```

Comment:
Worth noting that we use the workflow id to generate a workflow session object in local storage scoped by the id. This is a hack right now and we'll eventually want a better way for generating unique ids for workflows and workflow sessions.